intro-sql-access.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: 'Query your database using SQL'
+description: 'Run SQL queries over HTTP directly in PostgreSQL'
```

Comment:
Suggestion:

"Query PostgreSQL directly with SQL over HTTP in our REST API and SDKs"